Bottom: 12bca8e50774ee1f185378ff02152027cf0c7f0e
Top:    cf880b9fd7b1fec95c62cc71b52cd02d7f8553f5
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2015-01-08 21:29:21 +0000

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
